ninjalooter/raidgroups.py

```diff
@@ -0,0 +1,812 @@
+    master_player_dict['xWar60b']    = Player('xWar60b', 'Warrior', 60, 'fow')
+    master_player_dict['xWar59a']    = Player('xWar59a', 'Warrior', 59, 'fow')
+    master_player_dict['xWar59b']    = Player('xWar59b', 'Warrior', 59, 'fow')
+                                                
+    master_player_dict['xPal60']     = Player('xPal60', 'Paladin', 60, 'fow')
+    master_player_dict['xShd60']     = Player('xShd60', 'Shadow Knight', 60, 'fow')
+                                                
+    master_player_dict['xEnc60']     = Player('xEnc60', 'Enchanter', 60, 'fow')
+    master_player_dict['xEnc59']     = Player('xEnc59', 'Enchanter', 59, 'fow')
+    master_player_dict['xEnc58']     = Player('xEnc58', 'Enchanter', 58, 'fow')
+    master_player_dict['xMag60']     = Player('xMag60', 'Magician', 60, 'fow')
+    master_player_dict['xNec60']     = Player('xNec60', 'Necromancer', 60, 'fow')
+    master_player_dict['xWiz60']     = Player('xWiz60', 'Wizard', 60, 'fow')
+                                                
+    master_player_dict['xClr60']     = Player('xClr60', 'Cleric', 60, 'fow')
+    master_player_dict['xDru60']     = Player('xDru60', 'Druid', 60, 'fow')
+    master_player_dict['xShm60']     = Player('xShm60', 'Shaman', 60, 'fow')
+    master_player_dict['xShm60torp'] = Player('xShm60torp', 'Shaman', 60, 'fow')
+    master_player_dict['xShm59']     = Player('xShm59', 'Shaman', 59, 'fow')
+                                                
+    master_player_dict['xBrd60']     = Player('xBrd60', 'Bard', 60, 'fow')
+    master_player_dict['xBrd59']     = Player('xBrd59', 'Bard', 59, 'fow')
+    master_player_dict['xBrd57']     = Player('xBrd57', 'Bard', 57, 'fow')
+    master_player_dict['xMnk60']     = Player('xMnk60', 'Monk', 60, 'fow')
+    master_player_dict['xRng60']     = Player('xRng60', 'Ranger', 60, 'fow')
+    master_player_dict['xRog60']     = Player('xRog60', 'Rogue', 60, 'fow')
+
+
+    master_player_dict['aClr60']     = Player('aClr60', 'Cleric', 60, 'fow')
+    master_player_dict['bClr60']     = Player('bClr60', 'Cleric', 60, 'fow')
+    master_player_dict['cClr60']     = Player('cClr60', 'Cleric', 60, 'fow')
+    master_player_dict['dClr60']     = Player('dClr60', 'Cleric', 60, 'fow')
+    master_player_dict['eClr60']     = Player('eClr60', 'Cleric', 60, 'fow')
+    master_player_dict['fClr60']     = Player('fClr60', 'Cleric', 60, 'fow')
+    master_player_dict['gClr60']     = Player('gClr60', 'Cleric', 60, 'fow')
+    master_player_dict['hClr60']     = Player('hClr60', 'Cleric', 60, 'fow')
+    master_player_dict['iClr60']     = Player('iClr60', 'Cleric', 60, 'fow')
+    master_player_dict['jClr60']     = Player('jClr60', 'Cleric', 60, 'fow')
+
+
+    print(len(master_player_dict))
+
+    # test the GroupBuilder
+    gb = GroupBuilder()
+
+
+#    print(master_player_dict)
+
+#    # build a test group.  Just for testing, allow the list to exceed 6
```

Comment:
Generally don't leave large comment blocks of unused code -- is this necessary to keep / serving a purpose as documentation?